.travis.yml

```diff
@@ -35,7 +31,9 @@ before_install:
 install:
   # Customise the testing environment
   # ---------------------------------
+  - conda install --channel ioos pytest
   - conda install --channel ioos --file requirements.txt
-  - conda install --channel ioos nco pyncml pytest pynco
+  - conda install --channel ioos pyncml pynco
```

Comment:
I :+1:  to that but I am not the main maintainer here. What do you prefer @kwilcox?